spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -292,7 +294,20 @@ public SparkContext createSparkContext() {
     }
     if (System.getenv("SPARK_HOME") != null) {
       conf.setSparkHome(System.getenv("SPARK_HOME"));
+
+      // load configs from spark properties file
+      final String sparkPropertiesFile =
+          Utils.getDefaultPropertiesFile(JavaConversions.asScalaMap(System.getenv()));
+      logger.info("Loading spark properties file at " + sparkPropertiesFile);
+
+      final Map<String, String> sparkConfigs =
+          JavaConversions.asJavaMap(Utils.getPropertiesFromFile(sparkPropertiesFile));
+
+      for (Map.Entry<String, String> entry : sparkConfigs.entrySet()) {
+        conf.set(entry.getKey(), entry.getValue());
```

Comment:
Shouldn't the explicit config (from getProperty()) overrides the ones from the Spark Properties file?